### PR TITLE
chore(Tag): fix typescripts

### PR DESCRIPTION
**Pull request by youluna**

### CODE REVIEW

File: types/tag/index.d.ts [hunk 1]
```diff
@@ -4,9 +4,42 @@ import * as React from 'react';
 
 interface HTMLAttributesWeak extends React.HTMLAttributes<HTMLElement> {
     onClick?: any;
+    onChange?: any;
```
Comment (jdkahn) [inline types/tag/index.d.ts]: Only `Selectable` Tag has an `onChange` prop

File: types/tag/index.d.ts [hunk 2]
```diff
@@ -4,9 +4,42 @@ import * as React from 'react';
...
+
+    /**
+     * 标签出现动画结束后执行的回调
+     */
+    afterAppear?: () => void;
+
+    /**
+     * 点击回调
+     */
+    onClick?: () => void;
```
Comment (jdkahn) [inline types/tag/index.d.ts]: I think we can remove this.

File: types/tag/index.d.ts [hunk 3]
```diff
@@ -64,43 +93,18 @@ export interface SelectableProps extends HTMLAttributesWeak {
...
-     */
-    type?: 'normal' | 'primary';
 
-    /**
-     * 标签的尺寸（large 尺寸为兼容表单场景 large = medium）
-     */
-    size?: 'small' | 'medium' | 'large';
+export interface TagGroupProps extends React.HTMLAttributes<HTMLElement> {
+    prefix?: string;
+    rtl?: boolean;
```
Comment (jdkahn) [inline types/tag/index.d.ts]: Can we make a global interface for `rtl`? `Tag`, `ClosableTag`, and `TagGroup` all need and lots of other components do too.

**Comment by youluna** [comment]
fix in https://github.com/alibaba-fusion/next/pull/917

**Pull request closed without merging**